seqr/management/tests/check_for_new_samples_from_pipeline_tests.py

```diff
@@ -427,28 +429,31 @@ def test_command(self, mock_email, mock_temp_dir):
         })
 
         # Tests Sample models created/updated
-        updated_sample_models = Sample.objects.filter(guid__in=SAMPLE_GUIDS+GCNV_SAMPLE_GUIDS)
-        self.assertEqual(len(updated_sample_models), len(SAMPLE_GUIDS+GCNV_SAMPLE_GUIDS))
+        snv_indel_samples = Sample.objects.filter(data_source='auto__2023-08-09')
+        gcnv_samples = Sample.objects.filter(data_source='auto__2024-09-14')
+        updated_sample_models = snv_indel_samples | gcnv_samples
```

Comment:
I changed the logic in this part of the test because the way it was written if we alter the test so that more samples are created the test will succeed without testing those new samples. Changing the order here now causes the test to fail if you are creating more samples that are not added to `SAMPLE_GUIDS` or `GCNV_SAMPLE_GUIDS`